.vscode/launch.json

```diff
@@ -15,16 +15,6 @@
                 "--colors"
             ],
             "internalConsoleOptions": "openOnSessionStart"
-        },
-        {
```

Comment:
Did this script stop working and that's why it's going away?

Reply:
We've moved away from the test server. It was frequently failing in CI environment and it was hard to reproduce the errors so we changed the test code to mocks.

Reply:
That was some time ago, removing to keep things up-to-date.